modules/gltf/gltf_document.cpp

```diff
@@ -6625,3 +6689,15 @@ Error GLTFDocument::_serialize_file(Ref<GLTFState> state, const String p_path) {
 	}
 	return err;
 }
+void GLTFDocument::_normalize_bone(int32_t p_bone, Set<BoneId> &r_bones, float &r_joint, float &r_weight) {
+	if (!Math::is_zero_approx(r_weight)) {
```

Comment:
combine conditions to one
	
	if (!Math::is_zero_approx(r_weight) && !r_bones.has(p_bone)) {
	..
	else 
	r_weight = 0.0f;